test/test_myconverter.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from myconverter.converter import MyConverter
+import os
```

Comment:
As mentioned above, for code quality: Rearranging "import" portion into three separated parts.
    First: Native library.
    Second: Non-native library through package manager (ex: numpy, etc), locally installed library.
    Third: Function which imported from the same code base.